src/store/atoms.ts

```diff
@@ -0,0 +1,15 @@
+import formatCartItems from '@/services/formatCartItem';
+import { atom, selector } from 'recoil';
+import { fetchCartItems } from '../api';
+
+export const allCartItemStates = atom({
```

Comment:
[single sourth of truth](https://en.wikipedia.org/wiki/Single_source_of_truth) 👏